composer.json

```diff
@@ -97,7 +101,6 @@
 			"dev-main": "2.0.x-dev"
 		},
 		"typo3/cms": {
-			"app-dir": ".Build",
```

Comment:
I would leave this for typo3/cms-composer-installers v3 as v3 is used with TYPO3 v11 for now. See also docs for v3: https://github.com/TYPO3/CmsComposerInstallers/tree/3.1. For upcoming v4 (for now as RC available) "app-dir" was removed.